Builds/linux/README.md

```diff
@@ -239,3 +239,24 @@ change the `/opt/local` module path above to match your chosen installation pref
 `rippled` builds a set of unit tests into the server executable. To run these unit
 tests after building, pass the `--unittest` option to the compiled `rippled`
 executable. The executable will exit with summary info after running the unit tests.
+
+## Known Installation Issues
+
+*Compilation Errors pertaining to soci:* Certain compilation errors have been observed with Apple Clang 13.1.6+ and soci v4.x. soci compiles with -Werror flag which causes certain warnings to be reported as errors. These warings pertain to style and not that of correctness. Since these errors are more than 20 in number, it causes fatal termination of the cmake process.
```

Comment:
````suggestion
Compilation errors have been observed with Apple Clang 13.1.6+ and soci v4.x. soci compiles with the `-Werror` flag which causes warnings to be treated as errors. These warnings pertain to style (not correctness). However, they cause the cmake process to fail.

Here's an example of how this looks:
```
.../rippled/.nih_c/unix_makefiles/AppleClang_13.1.6.13160021/Debug/src/soci/src/core/session.cpp:450:66: note: in instantiation of function template specialization 'soci::use<std::string>' requested here
    return prepare << backEnd_->get_column_descriptions_query(), use(table_name, "t");
                                                                 ^
1 error generated.
```
````